README.md

```diff
@@ -10,6 +10,9 @@ from tripleo-quickstart here.
 # Pre-requisites
 
 - CentOS 7
```

Comment:
I would just say centos 7.5 here.  Maybe note that you must install as 7.4+ (IIRC), previous versions didn't set d_type when making the fs.

Reply:
How about both? This is a nasty one. Worth avoiding in the future.